### PR TITLE
Add compute scenarios

### PR DESCRIPTION
**Pull request by Alancere**
fixes: https://github.com/Azure/azure-sdk-for-go/issues/17884

### CODE REVIEW

**Review by tadelesh — APPROVED** [review]

**Comment by tadelesh** [comment]
@leni-msft  Could you help to merge this PR?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachineImagesEdgeZone.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/documentation/api-scenario/references/v1.2/schema.json
```
Comment (leni-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachineImagesEdgeZone.yaml:1]: delete this file?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachine.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        armTemplate: peArmTemplate/Create_NetworkInterface.json
+      - step: VirtualMachine_Create
+        exampleFile: ../examples/compute/CreateAWindowsVmWithPatchSettingAssessmentModeOfImageDefault.json
+        requestUpdate:
+          - replace: /parameters/properties/networkProfile/networkInterfaces/0/id
+            value: $(nicID)
+          - replace: /parameters/properties/osProfile/adminUsername
+            value: goscenario
+          - replace: /parameters/properties/osProfile/adminPassword
+            value: QWE!@#123
```
Comment (leni-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachine.yaml:28]: password is better to be `secureString` and pass in at runtime

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-12-01/scenarios/disk.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/documentation/api-scenario/references/v1.2/schema.json
+scope: ResourceGroup
+
+scenarios:
+  - scenario: ManagedDisk
+    description: Microsoft.Compute/disks CRUD
+    variables:
+      diskName: scenario_disk
```
Comment (leni-msft) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2021-12-01/scenarios/disk.yaml:8]: are these scenario files runnable?

Comment (Alancere) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2021-12-01/scenarios/disk.yaml:8]: runnable

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachineImagesEdgeZone.yaml (hunk 1 above)
Comment (Alancere) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachineImagesEdgeZone.yaml:1]: deleted

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachine.yaml (hunk 2 above)
Comment (Alancere) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/scenarios/virtualMachine.yaml:28]: Replace adminPwd with type secureString

**Pull request merged**